core/common/src/main/java/alluxio/RuntimeConstants.java

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
The constructor of `Configuration` (called in the static block here) makes use of many fields of `Constants`. Moving this static initialization block to `Constants` will make it impossible to statically initialize `Constants` without constructing a `Configuration`, and impossible to construct a `Configuration` without having `Constants` be statically initialized. To avoid this problem, we should make sure that `Constants` can be statically initialized without depending on any other classes.